packages/logger/lib/logger.js

```diff
@@ -338,6 +338,14 @@ class Logger {
 				sanitizedLogData.message = null;
 			}
 
+			if (sanitizedLogData.error && sanitizedLogData.error instanceof Error) {
+				sanitizedLogData.error = serializeError(sanitizedLogData.error);
+			}
+
+			if (sanitizedLogData.err && sanitizedLogData.err instanceof Error) {
+				sanitizedLogData.err = serializeError(sanitizedLogData.err);
+			}
```

Comment:
**issue:** I think let's remove this use-case from the PR as chatted about yesterday. We should also copy this explanation onto the issue as a comment. My original issue didn't consider the complexities of supporting this. We have two conflicting things which we need to think about more:

1. In the Splunk logs we want to be able to find error information easily regardless of the system. This means that ideally we should convert `err` to `error` so that our Splunk searches don't have to consider both, _however_...

2. If we convert `err` to `error` will that break existing dashboards that expect `err`? Also what happens if somebody logs both, which would be choose? e.g. `logger.error({ err: new Error('e1') }, new Error('e2'))`.

I think the right call is not to implement anything for this case. I don't think it's [super common](https://github.com/search?q=org%3AFinancial-Times+err%3A+language%3AJavaScript&type=code&l=JavaScript) for people to use `err`  and if we add this support in a way we're not happy with then we'll be stuck with it for ages. Let's be cautious.

Reply:
Ah yes, I recall our convo re: this. Ok will remove these lines